### DISCUSSION TITLE
Help for a Translation

### DISCUSSION THREAD

**Discussion by zoe838** [Monika-After-Story/MonikaModDev#8317] [general - translations]
Hi, I am new here and do not know, how all this here works. I would like to translate the Website in a other language, but I don´t know, how I could do that... If someone knows more about that, it would be nice to become a little help.

**Comment by ThePotatoGuy** [comment]
I don't see how helpful translating the website would be, as we don't post any news on the site. It's primary purpose is to link out to some social media, github, and downloading the mod.
